layouts/_default/baseof.html

```diff
@@ -41,6 +41,15 @@
     data-page-cat="{{ $pageCat }}"
     {{ block `body-attributes` . }}{{ end }}
   >
+    {{ if .Param "takeover-active" }}
+      {{ partial "tw/modal-takeover.html" ( dict
+        "filename" "2023/08/01jd-w2vj-77eb-3dp1.jpeg"
+        "hed" "Time is running out and we need your help"
+        "dek" "Only a few hours remain for your gift to Spotlight PA to be DOUBLED. Contribute now and ensure our trusted, nonpartisan investigative journalism can continue."
+        "link" "https://spotlightpa.fundjournalism.org/statecollege/?campaign=701Dn000000YqKfIAK"
```

Comment:
Did you see my comment before asking if this is the right donate link?

Reply:
This is the link provided to me.